README.md

```diff
@@ -462,3 +462,20 @@ Basically, given the `documentation` as a model, we do this:
 
 Dokka is built with Gradle. To build it, use `./gradlew build`.
 Alternatively, open the project directory in IntelliJ IDEA and use the IDE to build and run Dokka.
+
+Here's how to import and configure Dokka in IntelliJ IDEA:
+
+* Select "Import Project" from the IDEA welcome screen
```

Comment:
* Select "Open project"
* Select the directory with your clone of Dokka
* After IDEA opened project click `File > New > Module from existing sources` 
* Select `build.gradle` in the root of Dokka clone
* Use the default options and select "Finish"
>>> * After Dokka is loaded into IDEA.....

Reply:
Updated